tests/plotting/conftest.py

```diff
@@ -102,6 +102,14 @@ def colorful_tetrahedron():
     return mesh
 
 
+@pytest.fixture(autouse=True)
+def set_default_theme():
+    """Reset the testing theme for every test."""
```

Comment:
Can we get rid of the global theme handling at the very top of this file now?  It is better to have a single location.

Reply:
Totally agree, will do this